lebedigital/raw_data_processing/youngs_modulus_data/mixture_metadata_extraction.py

```diff
@@ -0,0 +1,283 @@
+# Script for metadata-extraction: 
```

Comment:
We should move the file to mixture (instead of Youngs modulus), because the mixture is not related to a specific (Youngs) modulus test.

Reply:
no, that has to be created